### ISSUE TITLE
Where is RunSFM_Ceres?

### ISSUE THREAD

**Issue by aki34** [snavely/bundler_sfm#61]
Hi
I wonder where RunSFM_Ceres is implemented.
According to header file(BundleApp.h), the function is not defined.
If I want to use Ceres solver, do I have to implement it?

**Comment by snavely** [comment]
No, you do not need to implement this function yourself. It is defined in BundleCeres.cpp. However, to compile with that functionality, you need to uncomment the "# USE_CERES=true" line in src/Makefile.

**Comment by aki34** [comment]
Can I use ceres by adding BundlerCeres.cpp to Source Files with Visual Studio 2013?

**Comment by snavely** [comment]
Yes, though you will also have to add the define __USE_CERES__ to the
build, and also tell the linker to link to the appropriate Ceres libraries.

**Comment by aki34** [comment]
In BundleCeres.cpp, sys/time.h is included but I can't use it in windows. So what should I do?

**Comment by aki34** [comment]
I can make time.h file from here.
https://stackoverflow.com/questions/10905892/equivalent-of-gettimeday-for-windows
By adding #undef ERROR to time.h after #include <windows.h>, I built bundler_sfm.
Thank you for your support.

**Comment by snavely** [comment]
Glad to hear it, and sorry for the difficulty compiling. Are you interested
in contributing your patch to the repository as a push request?

Noah

**Comment by aki34** [comment]
I sent pull request just now.
I'm not used to using git-hub so there can be some wrong but please check that.
